code/api/src/modules/user/resolvers.js

```diff
@@ -73,6 +66,16 @@ export async function remove(parentValue, { id }) {
   return await models.User.destroy({ where: { id } })
 }
 
+// email update
```

Comment:
This is the resolver which uses the mutation to edit a users' email address.

Reply:
should we rename this to `editEmail` to be more explicit?